test/phoenix/live_dashboard/helpers/reingold_tilford_test.ex

```diff
@@ -0,0 +1,186 @@
+defmodule Phoenix.LiveDashboard.ReingoldTilfordTest do
+  use ExUnit.Case, async: true
+  use PropCheck
```

Comment:
Those properties are amazing! Great job!

Quick question: how hard would it be to use [StreamData](https://github.com/whatyouhide/stream_data) instead? StreamData is smaller and can help us avoid any issues in regards to licenses. Proper is GPL licensed and given the dashboard is built-in into Phoenix apps, folks may want to avoid it - even in test.

Reply:
I understand!,  I will investigate how to generate the trees with StreamData.